federation/planner.go

```diff
@@ -145,22 +192,27 @@ func (e *Planner) planObject(typ *graphql.Object, selectionSet *graphql.Selectio
 			return nil, fmt.Errorf("typ %s has no field %s", typ.Name, selection.Name)
 		}
 
-		fieldInfo := e.schema.Fields[field]
+		var hasCustomService bool
```

Comment:
hasCustomService seems confusing - this seems to be false only if we see an error. Could we rename to err?

Reply:
I don't think it should be an error. 
This variable will be false for most of the time unless there is a migration going on.

Reply:
Ahh I see, I misunderstood what hasCustomService did